project-p3-forms-master/code/index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8" />
+    <title>Form for jobsurfer</title>
+    <link rel="stylesheet" href="style.css">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Handlee&display=swap" rel="stylesheet">
+  </head>
+  <body>
+
+    <form action="https://httpbin.org/anything" method="post">
+      <h3>Du möchtest Jobsurfer werden?</h3>
+      <h2>Erstelle dein Profil</h2>
+      <fieldset>
+        <label>
+          Vorname
+          <input type="text" name="vorname" required placeholder="Hans"/>
+        </label>
+        <label>
+          Name
+          <input type="text" name="name" required placeholder="Mustermann"/>
+        </label>
+        <label>
+          E-Mail
+          <input type="text" name="email" required placeholder="hans.mustermann@gmail.com"/>
+        </label>
+        <label>
+          Passwort
+          <input type="password" name="passwort" required placeholder="mindestens 8 Zeichen"/> <!-- Icon Auge einfügen -->
```

Comment:
I'm guessing the translation of this is "at least 8 characters" so I'm throwing in a link here ;) -> https://www.w3schools.com/tags/att_input_minlength.asp